frontend/styles/custom.css

```diff
@@ -33,19 +33,31 @@ header input {
 .bg-cite {
   background: url("/images/pexels-pixabay-159751_scaled.jpg")
 }
+
 .hgf-text-highlight {
   background-color: #cdeefb;
 }
-.hgf-scrollbar {
+
+#hgf-scroll-container {
+  display: flex;
+  height: 200px;
+}
+
+.simplebar-track.simplebar-horizontal {
+  height: 7px !important;
   border-bottom: 1px solid #002864;
```

Comment:
The bottom border was missing and I added it again. However, there is a 2 pixel offset at both ends. Is there any way to fix this?

![image](https://user-images.githubusercontent.com/14222414/191195564-16562953-9bf0-4412-8712-73914129b144.png)